linux/net/rina/dt-utils.c

```diff
@@ -953,7 +953,7 @@ int common_efcp_pdu_send(struct efcp * efcp,
 
 	/* Remote flow case */
 	if (pci_source(pci) != pci_destination(pci)) {
-	        if (rmt_send(rmt, pci, pdu)) {
```

Comment:
Yeah, there's no overhead in retrieving it once more. I prefer to keep the function signatures minimal.